pkg/pillar/hypervisor/kubevirt.go

```diff
@@ -691,6 +691,9 @@ func (ctx kubevirtContext) Info(domainName string) (int, types.SwState, error) {
 		res, err = getVMIStatus(vmis, nodeName)
 	}
 	if err != nil {
+		if strings.Contains(err.Error(), "etcdserver: request timed out") || strings.Contains(err.Error(), "apiserver not ready") {
```

Comment:
If getVMIStatus returns "Unknown", nil (instead of error) then you wouldn't need to duplicate the strings here.
Would "Unknown" magtch an entry in stateMap[]?

Reply:
> If getVMIStatus returns "Unknown", nil (instead of error) then you wouldn't need to duplicate the strings here. Would "Unknown" magtch an entry in stateMap[]?

Why would getVMIStatus returns "Unknown", nil (instead of error)  ? It's really an err right that we could not talk to k3s ? Yes "Unknown" is part of stateMap[]